style.css

```diff
@@ -0,0 +1,1330 @@
+/* Homepage */
+body {
+  width: 100%;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  margin: 0;
+}
+
+.active {
+  color: white;
+}
+
+nav .top {
+  display: flex;
+  justify-content: end;
+  gap: 1rem;
+  background: #3e3e3e;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  color: #fff;
+  height: 25px;
+  align-items: center;
+  width: 100%;
+}
+
+footer img,
+.about-footer img {
+  width: 250px;
+  height: 153px;
+}
+
+.entry-logo img {
+  width: 200px;
+  height: 200px;
+  padding-left: 600px;
+  margin-top: 0;
+  padding-top: 0;
+}
+
+nav .top img {
+  width: 10.12px;
+  height: 15px;
+}
+
+nav .top ul {
+  display: flex;
+  list-style: none;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  gap: 1rem;
+  padding-right: 63px;
+  padding-left: 0;
+}
+
+nav .bottom {
+  display: flex;
+  gap: 125px;
+  align-items: center;
+  height: 75px;
+  background: #fff;
+  padding-left: 0;
+  padding-right: 0;
+  width: 100%;
+}
+
```

Comment:
- [ Optional] You did well working on your navbar, I will just recommend adding `  justify-content: end;
`  so that your 'Compain' button will visible in all screen sizes and your nav bar will look perfect.